common/server/statshouse/add_metrics_batch.h

```diff
@@ -0,0 +1,32 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2021 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#pragma once
+
+#include <string>
+#include <vector>
+namespace statshouse {
+
+struct tl_query_payload {
+  int *buffer;
+  int len;
+};
+
+struct Metric {
```

Comment:
Naming `StatshouseMetric` instead of `Metric` is way better for text search / fuzzy search / method navigation

Reply:
done